### PR TITLE
remove inbox_watch and mvbox_watch options from advanced settings

### LINKED ISSUE #2176: Remove `inbox_watch` and `mvbox_watch` options from advanced settings
`mvbox_watch` is replaced with `mvbox_move`: https://github.com/deltachat/deltachat-core-rust/pull/2906
`inbox_watch` is removed: https://github.com/deltachat/deltachat-core-rust/pull/2922

### PR DESCRIPTION
**Pull request by adbenitez**
close #2176

### CODE REVIEW

**Comment by link2xt** [comment]
Should the translation sources be removed too? Or do we keep them because of desktop/ios still using them?

**Review by link2xt — APPROVED** [review]

**Comment by Hocuri** [comment]
We keep the translations because of desktop/ios still using them.

**Review by Hocuri — APPROVED** [review]

**Comment by r10s** [comment]
> Should the translation sources be removed too? Or do we keep them because of desktop/ios still using them?

i'd prefer to clean up later on in another pr.

File: src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java [hunk 1]
```diff
@@ -65,21 +61,11 @@ public void onCreate(Bundle paramBundle) {
...
     preferE2eeCheckbox.setOnPreferenceChangeListener(new PreferE2eeListener());
 
-    inboxWatchCheckbox = (CheckBoxPreference) this.findPreference("pref_inbox_watch");
-    inboxWatchCheckbox.setOnPreferenceChangeListener((preference, newValue) ->
-      handleImapCheck(preference, newValue, CONFIG_INBOX_WATCH)
-    );
-
     sentboxWatchCheckbox = (CheckBoxPreference) this.findPreference("pref_sentbox_watch");
     sentboxWatchCheckbox.setOnPreferenceChangeListener((preference, newValue) ->
       handleImapCheck(preference, newValue, CONFIG_SENTBOX_WATCH)
```
Comment (link2xt) [inline src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java:66]: I think this check is not needed, it warns me whenever I try to disable watching `Sent` that it is somehow dangerous and I should make sure it's the same on other devices etc. etc.

Comment (adbenitez) [reply src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java]: done

**Review by Hocuri — APPROVED** [review]

**Pull request merged**